### PR TITLE
Add varscan

### PR DESCRIPTION
**Pull request by shiltemann**
Please sign off by admins only
------------------------------

* [x] Has no duplicate (otherwise merge required)
* [ ] Indentation is correct
* [x] Tests
  - [x] Parameters are reasonably covered
  - [x] Test files are appropriate
* [ ] `<command/>`: file names are escaped
* [x] Tool requires version/build increase

### CODE REVIEW

File: tools/varscan_mpileup2snp_from_bam/test-data/generate_reads.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
```
Comment (shiltemann) [inline tools/varscan_mpileup2snp_from_bam/test-data/generate_reads.py]: flake8 this file

File: tools/varscan_mpileup2snp_from_bam/varscan_mpileup2snp_from_bam.xml [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+            </change_format>
+        </data>
+    </outputs>
+    
+    <tests>
+        <test><!-- Use classical samtools -->
+            <param name="alignments" value="example.bam" ftype="bam" />
+            
+            <param name="source_select" value="history" />
+            <param name="reference_genome" value="example.fa" ftypet="fasta" />
```
Comment (shiltemann) [inline tools/varscan_mpileup2snp_from_bam/varscan_mpileup2snp_from_bam.xml]: s/ftypet/ftype/

File: tools/varscan_mpileup2indel_from_bam/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+categories:
+- Variant Analysis
+- SAM
+description: VarScan2 mpileup2indel - optimized for direct BAM/SAM input and parallel mpileup generation
+long_description: |
+  VarScan is a variant caller for high-throughput sequencing data.
+name: varscan_mpileup2indel_from_bam
+owner: yhoogstrate
+remote_repository_url: https://bitbucket.org/EMCbioinf/galaxy-tool-shed-tools
```
Comment (shiltemann) [inline tools/varscan_mpileup2indel_from_bam/.shed.yml]: update repo url and give admin access

File: tools/varscan_mpileup2snp_from_bam/.shed.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+categories:
+- Variant Analysis
+- SAM
+description: VarScan2 mpileup2snp - optimized for direct BAM/SAM input and parallel mpileup generation
+long_description: |
+  VarScan is a variant caller for high-throughput sequencing data.
+name: varscan_mpileup2snp_from_bam
+owner: yhoogstrate
+remote_repository_url: https://bitbucket.org/EMCbioinf/galaxy-tool-shed-tools
```
Comment (shiltemann) [inline tools/varscan_mpileup2snp_from_bam/.shed.yml]: update repo url and give admin access

File: tools/varscan_mpileup2indel_from_bam/tool_dependencies.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="samtools" version="0.1.19">
```
Comment (yhoogstrate) [inline tools/varscan_mpileup2indel_from_bam/tool_dependencies.xml]: sambamba via conda?

File: tools/varscan_mpileup2indel_from_bam/varscan_mpileup2indel.xml [hunk 6]
```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tool id="varscan_mpileup2indel" name="VarScan2 Call INDELs from a mpileup file" version="2.3.6.a">
+    <description>VarScan2 INDEL detection; directly from a *.mpileup file.</description>
+    
+    <requirements>
+        <requirement type="package" version="2.3.6">varscan</requirement>
+    </requirements>
+    
+    <version_command>java -jar $JAVA_JAR_PATH/VarScan.v2.3.6.jar 2>&amp;1 | head -n 1</version_command>
```
Comment (yhoogstrate) [inline tools/varscan_mpileup2indel_from_bam/varscan_mpileup2indel.xml]: sambamba version at the end?

**Pull request merged**